tools/interfaces.md

```diff
@@ -1,5 +1,69 @@
-# Work in Progress
+Mozilla Firefox Data Analysis Tools
+-----------------------------------
 
-This document is a work in progress.
-The work is being tracked in
-[this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1341896).
+This is a starting point for making sense of (and gaining access to) all of the
+Firefox-related data analysis tools. There are a number of different tools
+available, all with their own strengths, tailored to a variety of use cases and
+skill sets.
+
+####[sql.telemetry.mozilla.org](stmo.md) (STMO)
+
+The [sql.telemetry.mozilla.org](https://sql.telemetry.mozilla.org) (STMO) site
+is an instance of the very fine [redash](https://redash.io/) software, allowing
+for SQL-based exploratory analysis and visualization / dashboard
+construction. Requires (surprise!)  familiarity with SQL, and for your data to
+be explicitly exposed as an STMO data source. Bugs or feature requests can be
+reported in our issue tracker.
+
+####[analysis.telemetry.mozilla.org](spark.md) (ATMO)
+
+The
+[analysis.telemetry.mozilla.org](https://https://analysis.telemetry.mozilla.org)
+(ATMO) site can be used to launch and gain access to virtual machines running
+Apache Spark clusters which have been preconfigured with access to the raw data
+stored in our long term storage (LTS) S3 buckets. Spark allows you to use
+Python or Scala to perform arbitrary analysis and generate arbitrary
+output. Once developed, a.t.m.o can also be used to run recurring Spark jobs
+for data transformation, processing, or reporting. Requires Python or Scala
+programming skills and knowledge of various data APIs. Learn more by visiting
+the documentation or tutorials.
```

Comment:
Can you link to [Custom Analysis with Spark](https://github.com/mozilla/firefox-data-docs/blob/master/tools/spark.md) or the [pyspark quickstart](https://spark.apache.org/docs/latest/quick-start.html)